### PR TITLE
Upgrade to k6 0.43.1, upgrade other dependencies

### PR DESCRIPTION
**Pull request by toscott**
What
Upgrade to k6 0.43.1, upgraded other dependencies with
```
go get -u ./...
```

Why
Wanted to try out the new built in browser capabilities with k6 0.43.X

How I tested
Ran both standard and docker builds. 
Used binary to run a headless browser test

Current docker container is not capable of running browser tests as it does not have a browser installed.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,35 +1,37 @@
 module github.com/grafana/xk6-output-influxdb
 
-go 1.18
+go 1.20
```
Comment (javaducky) [inline go.mod:3]: Drop back to 1.19. Some tooling does not support 1.20 yet, and we're keeping in step with k6 on this one.

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17-alpine as builder
+FROM golang:1.20-alpine as builder
```
Comment (javaducky) [inline Dockerfile:1]: Nice! Please update the destination version of Alpine to 3.17 (~line 7).

**Review by javaducky — CHANGES_REQUESTED** [review]
Couple minor issues which should solve build failure as well.

File: go.mod (hunk 1 above)
Comment (toscott) [reply go.mod]: Done

File: Dockerfile (hunk 2 above)
Comment (toscott) [reply Dockerfile]: Swapped out this one to golang:1.20-alpine3.17 to make it more obvious there is correlation with the second.

**Review by codebien — APPROVED** [review]
LGTM

**Review by javaducky — APPROVED** [review]
LGTM! Thank you! 💪

**Pull request merged**